accepted/2021/experimental-features/experimental-features.md

```diff
@@ -0,0 +1,380 @@
+# Experimental Features
+
+**PM** [Immo Landwerth](https://github.com/terrajobst)
+
+Starting with .NET 5, we've moved to fixed release schedule whereby we ship one
+version of .NET once a year, at RTM quality, with every other year being a long
+term support (LTS) release.
+
+This is in stark contrast to the .NET Framework days where we had multiple years
+for a given release, typically two to three years.
+
+Shipping more frequently has many advantages, such as being able to react
+quicker to trends and having an organization and customer base that is used to
+shipping & absorbing technology quicker to make the most of today's fast moving
+world.
+
+With .NET Core, we've solved the problem of the extremely high in-place
+compatibility bar that .NET Framework has. This reduces risks for unintentional
+behavioral changes and also allows certain parts of .NET to make deliberate
+breaking changes if necessary.
+
+As a result, this makes it viable to innovate across runtime, libraries, and
+languages, allowing us to build more compelling features that would have been
+almost impossible to do on .NET Framework. Examples include static linking,
+default interface members, and ref structs (such as `Span<T>`).
+
+However, it's sometimes challenging to deliver cross-cutting feature work in a
+single release, especially when they are breaking new ground in terms of
+expressiveness. A good example of such a feature is the upcoming improved
+numeric support in generic code, with the help of static interface members. Such
+a feature requires work in the runtimes, the languages, and the libraries. The
+work at the library level isn't very high, but it's all about finding the right
+set of support infrastructure so that developers are able to be productive at
+writing generic code that can rely on numerical expressions and operators. In a
+sense, the library work can't really fully start until the runtime and compiler
+support is sufficiently bootstrapped, but it's the most important part in the
+validation of the feature set. Practically, this means a good chunk of the
+feature will only be available halfway through a release cycle, which greatly
+reduces the time window for gathering feedback from other parts of the product
+as well as from customers using preview builds of the upcoming .NET version.
+
+Sadly, preview feedback isn't always sufficient to judge how well a feature
+works because
+
+1. Much fewer people use preview builds of .NET and
+2. Many people don't use .NET previews for their production projects, thus
+   greatly reducing the breadth & depth at which previews are being exercised.
```

Comment:
It's also difficult to explore the breadth of an impact to an ecosystem, e.g. library A depending on a feature, library B depending on library A and building on what it exposed, etc.